dist/src/test/java/io/camunda/zeebe/shared/management/ExportingEndpointTest.java

```diff
@@ -0,0 +1,210 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.shared.management;
+
+import static io.camunda.zeebe.shared.management.ExportingEndpointTest.RequestMatcher.requestTo;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Named.named;
+import static org.junit.jupiter.params.provider.Arguments.arguments;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import io.camunda.zeebe.gateway.impl.broker.BrokerClient;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerClusterState;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerClusterStateImpl;
+import io.camunda.zeebe.gateway.impl.broker.cluster.BrokerTopologyManager;
+import io.camunda.zeebe.gateway.impl.broker.request.BrokerRequest;
+import io.camunda.zeebe.shared.management.ExportingService.InvalidTopologyException;
+import java.util.Collection;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.ArgumentMatcher;
+import org.springframework.boot.actuate.endpoint.web.WebEndpointResponse;
+
+final class ExportingEndpointTest {
+
+  @ParameterizedTest
+  @MethodSource("validTopologies")
+  void shouldPauseOnAllBrokersAndPartitions(final BrokerClusterState topology) {
```

Comment:
:wrench: Related to above comment. This could be `ExportingServiceTest`. And we could add `ExportingEndpointTest` that only test two scenarios -> exportingService.pause returns success/failure. Later it would be good if we can add more specific error codes for different types of failure. Then we can extend this test to see if the exceptions are mapped to the correct error code.